public/index.html

```diff
@@ -4,17 +4,21 @@
     <title>Mozilla Mixed Reality Social Client</title>
     <script src="https://aframe.io/releases/0.7.0/aframe.js"></script>
     <script src="./app.bundle.js"></script>
+    <style>
+        .a-enter-vr {
+            top: 90px;
+            bottom: auto;
+        }
+    </style>
 </head>
 
 <body>
-    <a-scene
-        networked-scene="adapter: janus;
-                         room: 2;
+    <a-scene networked-scene="adapter: janus;
+                         room: 3;
                          serverURL: wss://quander.me:8989;
                          audio: true;
                          debug: true;
-                         connectOnLoad: false;"
-        mute-mic="eventSrc: a-scene; toggleEvents: action_mute">
+                         connectOnLoad: false;" mute-mic="eventSrc: a-scene; toggleEvents: action_mute">
```

Comment:
Still using prettier on the HTML? 
Also, why change the room number?

Reply:
+1 to either using a sane HTML formatter (not prettier) or leaving formatting as you found it.

Reply:
Yeah I should have used that instead. Also, I’m not sure what’s up with prettier. This file should be ignored. Maybe it’s VSCode’s default html format? I have everything set to format on save for prettier but I should probably make sure that ignores html.  I’ll look into it when I get back.